File: 02_representation/base64url/main.go
package main

import (
	"encoding/base64"
	"fmt"
)

func main() {
	text := "Hello world, mac was here"
	bytes := []byte(text)
	b64 := base64.RawURLEncoding.EncodeToString(bytes)
	fmt.Println(b64)
}

File: 02_representation/base64url/main.java
import java.nio.charset.StandardCharsets;
import java.util.Base64;

class Main {
    public static void main(String[] args) {
        String text = "Hello world, mac was here";
        byte[] bytes = text.getBytes(StandardCharsets.UTF_8);
        String b64 = Base64.getUrlEncoder().withoutPadding().encodeToString(bytes);
        System.out.println(b64);
    }
}
